tst/com/amazon/corretto/crypto/provider/test/LibCryptoRngTest.java

```diff
@@ -152,14 +152,22 @@ public void largeRequest() {
         rnd.nextBytes(bytes);
         // Ensure that the resulting bytes haven't been left at zero
         // Probablistically, this will pass.
-        byte[] tests = new byte[3];
-        tests[0] = bytes[8192];
-        tests[1] = bytes[8193];
-        tests[2] = bytes[12287];
-        rnd.nextBytes(bytes);
-        assertTrue(tests[0] != bytes[8192]);
-        assertTrue(tests[1] != bytes[8193]);
-        assertTrue(tests[2] != bytes[12287]);
+        final int retries = 5;
+        for (int ii = 0; ii < retries; ii++) {
```

Comment:
Is there a reason to use `ii` instead of `i`?

Reply:
habit. it's easier to search on in large loops where word-searching (`*` in normal mode of vim for me, your editor may use something else) often false-positive matches in comments, etc. where the sequence ` i ` may be present.